data/io.elementary.files.appdata.xml.in.in

```diff
@@ -56,6 +56,23 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="6.0.1" date="2021-07-15" urgency="medium">
+      <description>
+        <p>Fixes:</p>
+        <ul>
+          <li>Fix truncation of slot if reclicked in Column View</li>
+        </ul>
+        <p>Improvements:</p>
+        <ul>
+          <li>Show more keyboard accelerators in menus</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Now requires granite version >= 6.1.0</li>
```

Comment:
I'm not sure that this is a user-facing change. Is there a user-facing benefit about this we can list?

Reply:
The benefit at the moment is the ability to put keyboard accelerators in the tab context menus - which is already covered elsewhere.  I'll remove it.  I guess any user who wants to compile from source will find this out from the README file.